pkg/model/sdk_helper.go

```diff
@@ -21,12 +21,14 @@ import (
 	"path/filepath"
 	"sort"
 	"strings"
+	"encoding/json"
```

Comment:
Please run `go fmt` on your code or add an auto-formatter to your IDE. It will automatically correct the above into an alphabetical import order.

Reply:
> add an auto-formatter to your IDE

I should really, always forget to run the formatter. Ran it manually on the two files for now.